pysat/utils/_core.py

```diff
@@ -136,6 +136,8 @@ def listify(iterable):
         list_iter = [arr_iter.tolist()]
     elif arr_iter.shape[0] >= 1:
         list_iter = arr_iter.flatten().tolist()
+    elif arr_iter.shape[0] == 0:
+        list_iter = arr_iter.tolist()
```

Comment:
Is there any other condition that could be triggered here?  If not, the `elif` above should just be `else`.  If there is, let's add an error check here that is triggered for `else`.

Reply:
Updated.